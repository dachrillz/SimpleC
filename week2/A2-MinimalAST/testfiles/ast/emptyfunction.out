Program
  List
    Function ID="a"
      null
      List
        Return
