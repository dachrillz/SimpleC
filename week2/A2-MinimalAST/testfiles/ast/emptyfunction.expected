Program
  List
    Function ID="a"
      List
        Return
